### ISSUE TITLE
create recurring tasks

### ISSUE THREAD

**Issue by xpdavid** [CS2103AUG2016-W09-C4/main#30]

**Comment by haiilong** [comment]
I've been thinking and poking around with this. It's actually cannot be done without major change in our code. Why?

1. What is recurring Task? Is it defined every week? every year? until when? Then does it have start time and end time instead of like "every day". Our Task class does not have this operator.

2. Our 'Add' command also does not support this. We either need 1 more assignment like ' /r isRecurring', do we need to make sure startTime and endTime must be contained in one day??? This will break EVERYTHING.
Or we have a new class: "Add recurring" - 'addr' seperately that takes in '/r isRecurring' and '/f frequency' that will call the add function repeatedly.

3. But if we do that..... what happens when we edit one of them, how can we update many others? Impossible. Or what happen to our list?

If we want to implement this, we need our Task to have isRecurring(), and we potentially need a seperate class to handle this recurring thing alone.

In conclusion: screw this. Jim's tasks come in emails anyways, we don't want him with this unnecessary features. HAHAHAHAHAHA
j/k j/k: open for discussion tomorrow. :)

**Comment by JasonScourge** [comment]
The issue is how are we going to show it will "recur" again from time to time after you completed it?
How is it going to do show if it is is like "done" for today, then tomorrow it resets as one of the few tasks again.

**Comment by xpdavid** [comment]
I think we don't need to do this. The auto resetting makes a little sense to me.

**Comment by JasonScourge** [comment]
#105

**Comment by xpdavid** [comment]
fixed in #105
